packages/fast-components-react-msft/src/action-toggle/action-toggle.spec.tsx

```diff
@@ -0,0 +1,291 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16/build";
+import { configure, mount, shallow } from "enzyme";
+import examples from "./examples.data";
+import MSFTActionToggle, {
+    ActionToggleAppearance,
+    ActionToggleHandledProps,
+    ActionToggleManagedClasses,
+    ActionToggleProps,
+    ActionToggleUnhandledProps,
+} from "./action-toggle";
+import { ActionToggle } from "./index";
+import { ActionToggleClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/*
+ * Configure Enzyme
+ */
+configure({ adapter: new Adapter() });
+
+describe("action toggle", (): void => {
+    const managedClasses: ActionToggleClassNameContract = {
+        actionToggle: "action-toggle",
+        actionToggle_selectedGlyph: "selectedGlyph",
+        actionToggle_unselectedGlyph: "unselectedGlyph",
+        actionToggle__primary: "action-toggle-primary",
+        actionToggle__lightweight: "action-toggle-lightweight",
+        actionToggle__disabled: "action-toggle-disabled",
+        actionToggle__selected: "action-toggle-selected",
+        actionToggle__hasGlyphAndContent: "action-toggle-hasGlyphAndContent",
+    };
+    const href: string = "#";
+
+    test("should have a displayName that matches the component name", () => {
+        expect((MSFTActionToggle as any).name).toBe(MSFTActionToggle.displayName);
+    });
+
+    test("should not throw if managedClasses are not provided", () => {
+        expect(() => {
+            shallow(
+                <MSFTActionToggle selectedLabel="selected" unselectedLabel="unselected" />
+            );
+        }).not.toThrow();
+    });
+
+    test("should create unselectedglyph element passed in", () => {
```

Comment:
typo I think "...element _if_ passed in" 
This occurrence is elsewhere in the file.

Reply:
Not seeing resolution or reasoning for not resolving

Reply:
changes uploaded